examples/diabetes_example/run_diabetes.py

```diff
@@ -0,0 +1,506 @@
+"""
+===========================================
+Federated Analytics on the diabetes dataset
+===========================================
+
+This example demonstrates how to use the flexibility of the Substra library to do Federated Analytics.
+
+We use the **Diabetes dataset** available from the `Scikit-Learn dataset module <https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset>`__.
+This dataset contains medical information such as Age, Sex or Blood pressure.
+The goal of this example is to compute some analytics such as Age mean, Blood pressure standard deviation or Sex percentage.
+
+We simulate having two different data organisations, and a third organisation which wants to access aggregated analytics
+without having access to the raw data. The example here runs everything locally; however there is only one parameter to
+change to run it on a real network.
+
+**Caution:**
+ This example is provided as an illustrative example only. In real life, you should be carefully not to
+ accidentally leak private information when doing Federated Analytics. For example if a column contains very similar values,
+ sharing its mean and its standard deviation is functionally equivalent to sharing the content of the column.
+ It is **strongly recommended** to consider what are the potential security risks in your use case, and to act accordingly.
+ It is possible to use other privacy-preserving techniques, such as
+ `Differential Privacy <https://en.wikipedia.org/wiki/Differential_privacy>`_, in addition to Substra.
```

Comment:
I'm struggling to find a better wording than "in addition to". 
I would like to highlight that both approaches are complementing one another and that it makes sense to combine both.

Reply:
"to complement what Substra provide..." ?